Lib/gftools/builder/operations/fontc/__init__.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from typing import List
+from gftools.builder.operations import OperationBase
+
+FONTC_PATH = "FONTC_PATH_NOT_SET"
+
+
+# should only be called once, from main, before doing anything else. This is a
+# relatively non-invasive way to smuggle this value into FontcOperationBase
+def set_global_fontc_path(path: Path):
+    global FONTC_PATH
+    FONTC_PATH = path
```

Comment:
This should fail if the FONTC_PATH is set already and fail if path doesn't point to a file. Maybe even fail if it doesn't point to something executable.

Reply:
I tried doing this logic and then python had some error about setting a global twice? so I think that's actually baked in now?

the check that the file exists happens earlier, at the call site.

Reply:
Hm, I don't remember there being any rules about multiple writes to globals.

I just tried locally and python happily let me write to a global repeatedly.